examples/src/main/java/io/kubernetes/client/examples/PatchExample.java

```diff
@@ -102,17 +101,17 @@ public static void main(String[] args) throws IOException {
       // apply-yaml a deployment, server side apply is alpha in kubernetes v1.14,
       // You need to actively enable the Server Side Apply alpha feature
       // https://kubernetes.io/docs/reference/using-api/api-concepts/#server-side-apply
```

Comment:
Is this comment is still needed given in the lite of 1.18?

Reply:
good catch, i updated the comment to clarify that server-side-apply is enabled by default after v1.16